spec/syntax.md

```diff
@@ -834,14 +840,14 @@ _option values_, and _keys_)
 _Messages_ that contain right-to-left (aka RTL) characters SHOULD use one of the 
 following mechanisms to make messages display intelligibly in plain-text editors:
```

Comment:
Before getting much deeper into discussing the mechanisms, could you clarify if this is intended to be the canonical/recommended way of isolating or marking messages with RTL contents that we've discussed, or is that something that'll be provided separately?

Reply:
No, that's separate.